docs/Es6ClassMocks.md

```diff
@@ -349,7 +349,7 @@ jest.mock('./sound-player', () => {
 });
 ```
 
-This will let us inspect usage of our mocked class, using `SoundPlayer.mock.calls`: `expect(SoundPlayer).toHaveBeenCalled();` or near-equivalent: `expect(SoundPlayer.mock.calls.length).toBe(1);`
+This will let us inspect usage of our mocked class, using `SoundPlayer.mock.calls`: `expect(SoundPlayer).toHaveBeenCalled();` or near-equivalent: `expect(SoundPlayer.mock.calls.length).toBeGreaterThan(0);`
```

Comment:
In parity with Mock Functions page:

https://github.com/facebook/jest/blob/50bab21a8e88215061228e4197bfe632a529692c/docs/MockFunctions.md?plain=1#L285-L286

https://github.com/facebook/jest/blob/50bab21a8e88215061228e4197bfe632a529692c/docs/MockFunctions.md?plain=1#L301-L302